### PR TITLE
Update some of the explanatory text

### PR DESCRIPTION
**Pull request by kirbyju**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

File: modules/tcia_dataset.ipynb [hunk 1]
```diff
@@ -1,728 +1,863 @@
 {
```
Comment (wyli) [inline modules/tcia_dataset.ipynb:1]: <p>thanks Justin, please clear this output, and the rest of the notebook updates look great.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Comment by Nic-Ma** [comment]
I see this PR updated the `TCIA` tutorial under `/modules/` and also added the same one under the root folder.
Seems like unexpected or mistake?
CC @wyli 

Thanks.

**Comment by kirbyju** [comment]
Yes, it was meant to go into /modules/.  Hopefully I've fixed the issue now.

**Comment by wyli** [comment]
thanks, there are some minor style issues that couldn't be fixed automatically: https://github.com/Project-MONAI/tutorials/runs/7995607531?check_suite_focus=true#step:7:76

**Review by wyli — APPROVED** [review]
Thanks!

**Pull request merged**